cmd/waku/server/rest/waku_rest.go

```diff
@@ -72,6 +77,7 @@ func (r *WakuRest) Start(ctx context.Context, wg *sync.WaitGroup) {
 
 	if r.node.Relay() != nil {
 		go r.relayService.Start(ctx)
+		go r.filterService.Start(ctx)
```

Comment:
Shouldn't this go in line 69 instead of here? looking at filter service implementation i don't see it using Relay()